src/status_im/contexts/chat/home/new_chat/view.cljs

```diff
@@ -136,8 +133,8 @@
          :render-section-header-fn contact-list/contacts-section-header
          :render-section-footer-fn contact-list/contacts-section-footer
          :content-container-style  {:padding-bottom 70}
-         :render-fn                render-fn
-         :scroll-enabled           @scroll-enabled?
+         :render-fn                contact-item-render
+         :render-data              {:set-has-error set-has-error}
```

Comment:
This is exactly the fix I found here https://github.com/status-im/status-mobile/issues/21658#issuecomment-2492104289 coincidentally. Thanks for the fix @seanstrom

Reply:
And the same I'm applying in my WIP for:
- https://github.com/status-im/status-mobile/issues/21658

@seanstrom Are you fixing all the avatar usages in this PR or just this one? I'm asking because we don't want to duplicate work

Reply:
This PR is only intending to fix the usage of the reloading of the avatar images inside the contact-list view for a new-chat, but it's possible that some of the changes related to the image source uri will affect other usages too.

Reply:
Got it, then I'll continue with

- #21658 

Once this PR is merged. Thanks for all the fixes